python/components/deframers/geoscan_deframer.py

```diff
@@ -45,10 +45,15 @@ def __init__(self, syncword_threshold=None, options=None):
 
         if syncword_threshold is None:
             syncword_threshold = self.options.syncword_threshold
+        
+        if new_geoscan_platform:
+            self.packlen=74
+        else:
+            self.packlen=66
```

Comment:
I'm wondering if instead of having a boolean `new_geoscan_platform` parameter we should have a `packlen` parameter. This seems better moving forward in case the packet length is changed yet again.

Also, there is no reason to create a class member `self.packlen` here. This could have been assign to a local variable `packlen` instead.